public/app/features/dashboard-scene/serialization/DashboardSceneSerializer.test.ts

```diff
@@ -565,9 +565,47 @@ describe('DashboardSceneSerializer', () => {
       });
     });
 
-    it('should throw on getTrackingInformation', () => {
-      const serializer = new V2DashboardSerializer();
-      expect(() => serializer.getTrackingInformation()).toThrow('Method not implemented.');
+    describe('tracking information', () => {
+      it('provides dashboard tracking information with no initial save model', () => {
+        const dashboard = setupV2();
+        const serializer = new V2DashboardSerializer();
+        expect(serializer.getTrackingInformation(dashboard)).toBe(undefined);
+      });
+
+      it('provides dashboard tracking information with from initial save model', () => {
+        const dashboard = setupV2({
+          timeSettings: {
+            nowDelay: '10s',
+            from: '',
+            to: '',
+            autoRefresh: '',
+            autoRefreshIntervals: [],
+            quickRanges: [],
+            hideTimepicker: false,
+            weekStart: '',
+            fiscalYearStartMonth: 0,
+            timezone: '',
+          },
+          liveNow: true,
+        });
+
+        const serializer = new V2DashboardSerializer();
+        serializer.initialSaveModel = dashboard.getSaveModel()! as DashboardV2Spec;
```

Comment:
wonder  - why are you setting this up, while everything is already set up, and coule be just trested by `dashboard.getTrackingInformation()` call?